STROOP/Structs/ObjectAssociations.cs

```diff
@@ -295,9 +295,9 @@ public uint AlignJPBehavior(uint segmented)
             // Unload and dispose of all images
             foreach (var obj in _objAssoc)
             {
-                obj.Image?.Image?.Dispose();
-                obj.TransparentImage?.Image?.Dispose();
-                obj.MapImage?.Image?.Dispose();
+                obj.Image?.Dispose();
```

Comment:
Yoshi is going to beat you to the fix in #113 . So you can revert these changes. Thanks for fixing it though.